Gemfile.lock.release

```diff
@@ -466,45 +472,45 @@ GEM
       typhoeus (~> 1.0, >= 1.0.1)
     awesome_spawn (1.6.0)
     aws-eventstream (1.3.0)
-    aws-partitions (1.894.0)
-    aws-sdk-cloudformation (1.99.0)
-      aws-sdk-core (~> 3, >= 3.191.0)
+    aws-partitions (1.933.0)
```

Comment:
lol, aws versioning... we're jumping 39 minor versions!  🤣

Reply:
Semver FTW